.buildkite/hooks/pre-exit

```diff
@@ -14,7 +14,7 @@ if [ -n "$TEST_INTEG_AUTH_GCP_SERVICE_TOKEN_FILE" ]; then
   fi
 fi
 
-unset GOOGLE_APPLICATION_GCP_SECRET API_KEY_TOKEN
+sh .buildkite/hooks/unset-secrets.sh
```

Comment:
```suggestion
source .buildkite/scripts/unset-secrets.sh
```
The script isn't present in `.buildkite/hooks`.

Reply:
Oh!